src/main.cpp

```diff
@@ -66,6 +106,38 @@ void pollEvents(sf::Window& out_window, bool& out_bQuit) {
 } // namespace
 
 int main(int argc, char** argv) {
+
+	int intNumber, base;
+	string binaryNumber;
+
+	// cout << "Usage: <type(f/i)> <value> <base(10/2)\n";
+	cout << "Enter the base of the number you want to convert <2> or <10>: ";
+	cin >> base;
+
+	if (base == 10) {
+
+		cout << "Enter the number you wish to convert. It must be a positive  integer: ";
+		cin >> intNumber;
+		if (intNumber < 0) {
+			cout << "Number must be positive";
+			return 0;
+		}
+		decimalToBinary(intNumber);
+	} else if (base == 2) {
+		cout << "Enter your binary number without any spaces. ";
+		cin >> binaryNumber;
+		if (!checkBinary(binaryNumber)) {
+			cout << "The given binary number must consist of 1 and 0\n";
+			return 0;
+		}
+
+		binaryToDecimal(binaryNumber);
+	}
+
+	else {
+		cout << "Error: You must input 2 or l0!";
+		return 0;
```

Comment:
`return 1` or anything non-zero (assume 0-255 range only, remaining codes are reserved for OS use) on failure